neuraxle/steps/output_handlers.py

```diff
@@ -19,10 +19,42 @@
     limitations under the License.
 
 """
-from neuraxle.base import ExecutionContext, BaseStep
+from neuraxle.base import ExecutionContext, BaseStep, NonFittableMixin, MetaStepMixin
 from neuraxle.data_container import DataContainer
 
 
+class OutputTransformerWrapper(NonFittableMixin, MetaStepMixin, BaseStep):
```

Comment:
Remove `NonFittableMixin` from here. I think this could be fittable (e.g.: imagine a label normalizer that must learn how to normalize labels).

Reply:
Please also rename the `OutputTransformer*Mixin*` to `InputAndOutputTransformer*Mixin*` since our mixin tuples them and does not only change the output.

Reply:
Implemented handle_fit, handle_fit_transform, and fit in OutputTransformerWrapper, renamed OutputTransformerMixin to InputAndOutputTransformerMixin